<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>total_incomes</class>
 <widget class="QMainWindow" name="total_incomes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #aaaaaa;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>511</width>
      <height>221</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>source</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>amount</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>description</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>date</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #00ffff;             
	text-decoration: underline;
	font: 700 italic 9pt &quot;Arial&quot;;
    font-size: 30px;                
    font-weight: bold;        
    letter-spacing: 2px;          
    text-align: center;              
    background-color: transparent;    

}
</string>
    </property>
    <property name="text">
     <string>INCOMES REPORT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>430</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 700 9pt &quot;Arial&quot;;
    color: #000000;                 
	font-size: 18px;             
    font-weight: bold;       
	background-color:transparent;       
}</string>
    </property>
    <property name="text">
     <string>Total income From all Sources :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTotalAll">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>420</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="totalincomeBack">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 700 italic 9pt &quot;Arial&quot;;
    background-color: #55aaff;
    border: 1px solid #003300;
    border-radius: 5px;
    padding: 5px;
    font-size: 14px;
    color: #000000;
}
QPushButton:hover {
    background-color: #aaffff;
}
</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 700 9pt &quot;Arial&quot;;
    color: #000000;                 
	font-size: 18px;             
    font-weight: bold;       
	background-color:transparent;       
}</string>
    </property>
    <property name="text">
     <string>Select the month you want to view:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxMonth">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>100</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Select a Month</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>January</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>February</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>March</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>April</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>May</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>June</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>July</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>August</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>September</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>October</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>November</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>December</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="showDetailsButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>160</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 700 italic 9pt &quot;Arial&quot;;
    background-color: #aaaaff;
    border: 1px solid #003300;
    border-radius: 5px;
    padding: 5px;
    font-size: 14px;
    color: #003300;
}
QPushButton:hover {
    background-color: #66cc66;
}
</string>
    </property>
    <property name="text">
     <string>Show Details</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>220</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 700 9pt &quot;Arial&quot;;
    color: #000000;                 
	font-size: 15px;             
    font-weight: bold;       
	background-color:transparent;       
}</string>
    </property>
    <property name="text">
     <string>Total income from Job :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>260</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 700 9pt &quot;Arial&quot;;
    color: #000000;                 
	font-size: 15px;             
    font-weight: bold;       
	background-color:transparent;       
}</string>
    </property>
    <property name="text">
     <string>Total income from Allownce from  parents :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>300</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 700 9pt &quot;Arial&quot;;
    color: #000000;                 
	font-size: 15px;             
    font-weight: bold;       
	background-color:transparent;       
}</string>
    </property>
    <property name="text">
     <string>Total income from schorlaships :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>340</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 700 9pt &quot;Arial&quot;;
    color: #000000;                 
	font-size: 15px;             
    font-weight: bold;       
	background-color:transparent;       
}</string>
    </property>
    <property name="text">
     <string>Total income from student aids :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>380</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 700 9pt &quot;Arial&quot;;
    color: #000000;                 
	font-size: 15px;             
    font-weight: bold;       
	background-color:transparent;       
}</string>
    </property>
    <property name="text">
     <string>Total income from others:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAllowance">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>250</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditStudentAid">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>330</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOthers">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>370</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditScholarship">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>290</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditJob">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>210</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
